ext/datasource/etcdv3/etcdv3.go

```diff
@@ -0,0 +1,112 @@
+package etcdv3
```

Comment:
Maybe we should add some unit test for Etcdv3DataSource

Reply:
accept

Reply:
I googled some information and found NOT mock etcd server.
Some integration test case is enough.

Reply:
accept